lib/plugins/create/create.js

```diff
@@ -223,7 +223,7 @@ class Create {
 
     // create (if not yet present) and chdir into the directory for the service
     if (boilerplatePath) {
-      const newPath = path.join(process.cwd(), boilerplatePath);
+      const newPath = path.resolve(boilerplatePath);
```

Comment:
```suggestion
      const newPath = path.resolve(process.cwd(), untildify(boilerplatePath));
```

Reply:
ok. i will change that. but i definitely tested ~/

Reply:
`path.resolve` doesn't expand `~/` you may simply confirm it in Node.js console.

If it worked for you it means some other code path was followed, where `untildify` was involved

Reply:
got u. could u pls check if we can merge this pr now ?